include/glow/Importer/ProtobufLoader.h

```diff
@@ -103,15 +103,36 @@ class ProtobufLoader {
   /// \returns the tensor that was registered under the name \p name.
   Tensor *getTensorByName(const std::string &name);
 
+  /// Create a new node \p name initialized with \p tensor.
+  /// The created node is also registered under \p name.
+  /// \returns the newly created node.
+  /// \pre !hasNodeByName(name)
+  Node *createAndRememberNode(
+      const std::string &name, Tensor &tensor,
```

Comment:
Can we use StringRef here?

Reply:
Good catch.
I believe so.
I mechanical restructured the code, I didn't try to improve the API.
Same question as the other comment should we do it in this PR or as a follow-up?
(I tend to prefer PR with one commit one intent: refactoring vs. API clean-up.)